gubernator/templates/filtered_log.html

```diff
@@ -7,21 +7,34 @@
 	<h1>{% if pr %}<a href="/pr/{{pr}}">PR #{{pr}}</a> {% endif %}<a href="/builds{{job_dir}}">{{job}}</a> <a href="/build{{build_dir}}">#{{build}}</a> {{log_file}}</h1>
 </div>
 <div id="failures">
+	<h3> Failed pod: {{pod}} </h3>
 	<form method="get" onchange="submit()">
 		<input type="hidden" name="pod" value="{{pod}}">
 		<input type="hidden" name="junit" value="{{junit}}">
 		<hr>
 		<label><input type="checkbox" name="wrap"{% if wrap %} checked{% endif %}> Line wrapping on</label>
+		% for folder, filenames in all_logs.items()|sort
+			<hr>
+			% if folder|basename == ""
+			% set artifact = folder|dirname|basename
+			% else
+			% set artifact = folder|basename
+			% endif
```

Comment:
Is there a way to move some of this logic out of jinja? E.g. by moving it into the `all_logs` function, and having that return more values? Ditto for the filename logic below.